File: fun/luas_persegi.py
def luas_default_tanpa_parameter_tanpa_return():
    sisi = 4
    luas = sisi * sisi
    print(f"Luas persegi dengan sisi {sisi} adalah {luas}")
def luas_default_tanpa_parameter_dengan_return():
    sisi = 5
    luas = sisi * sisi
    return luas
def hitung_luas_tanpa_return(sisi):
    luas = sisi * sisi
    print(f"Luas persegi dengan sisi {sisi} adalah {luas}")
def hitung_luas_dengan_return(sisi):
    luas = sisi * sisi
    return luas
luas_default_tanpa_parameter_tanpa_return()

hasil1 = luas_default_tanpa_parameter_dengan_return()
print(f"Luas persegi default adalah {hasil1}")

hitung_luas_tanpa_return(6)

hasil2 = hitung_luas_dengan_return(8)
print(f"Luas persegi adalah {hasil2}")

File: fun/luas_persegi.dart
void main() {
  luasDefaultTanpaParameterTanpaReturn();

  int hasil1 = luasDefaultTanpaParameterDenganReturn();
  print("Luas persegi default adalah $hasil1");

  hitungLuasTanpaReturn(6);

  int hasil2 = hitungLuasDenganReturn(8);
  print("Luas persegi adalah $hasil2");
}
void luasDefaultTanpaParameterTanpaReturn() {
  int sisi = 4;
  int luas = sisi * sisi;

  print("Luas persegi dengan sisi $sisi adalah $luas");
}
int luasDefaultTanpaParameterDenganReturn() {
  int sisi = 5;
  int luas = sisi * sisi;

  return luas;
}
void hitungLuasTanpaReturn(int sisi) {
  int luas = sisi * sisi;

  print("Luas persegi dengan sisi $sisi adalah $luas");
}
int hitungLuasDenganReturn(int sisi) {
  return sisi * sisi;
}